cypress/integration/components/Tools/Buttons/EditorButton.spec.js

```diff
@@ -0,0 +1,18 @@
+/// <reference types="cypress" />
+
+context('Click Editor Button Go to Editor page and return to home page', () => {
```

Comment:
Was this file included by accident? It should not be part of this PR. Can you please exclude this file from the PR?

Reply:
Yes! I will fix.

Reply:
done